### PR TITLE
List toolbar search actions

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: src/app/visual/list-toolbar/list-toolbar-visual.component.html [hunk 1]
```diff
@@ -326,3 +326,26 @@ <h2>Toolbar with multiselect (based on item.isSelected property)</h2>
...
+  <sky-list>
+    <sky-list-toolbar
+      toolbarType="search"
+    >
+      <sky-list-toolbar-search-actions>
+        <button
+          class="sky-btn sky-btn-primary"
+          type="button"
+        >
+          Add
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/list-toolbar/list-toolbar-visual.component.html]: Maybe not a big deal, but the focus on this button gets cut off by the search bar.

Screen Shot 2019-07-01 at 11 25 20 AM

File: src/app/public/modules/list-toolbar/list-toolbar.component.scss [hunk 2]
```diff
@@ -1,12 +1,57 @@
 @import "~@skyux/theme/scss/mixins";
 
-.sky-list-toolbar-search {
-  ::ng-deep .sky-toolbar-container, ::ng-deep .sky-search-dismiss-absolute {
-    background-color: $sky-background-color-neutral-light;
+.sky-list-toolbar-search-wrapper {
+  display: flex;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-toolbar/list-toolbar.component.scss]: While in responsive mode, click the magnifying glass button to expand the search, and you'll see this:

Screen Shot 2019-07-01 at 11 27 47 AM

The master branch doesn't look pretty either, but this is a pretty significant difference. Might be considered breaking. I'm guessing it has something to do with the new host tag combined with flex styles?

File: src/app/visual/list-toolbar/list-toolbar-visual.component.html (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/visual/list-toolbar/list-toolbar-visual.component.html]: Done!

File: src/app/public/modules/list-toolbar/list-toolbar.component.scss (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/list-toolbar/list-toolbar.component.scss]: Done!

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-ToddRoberts You are correct: we will not be documenting this feature, officially.

File: src/app/public/modules/list-toolbar/list-toolbar-search-actions.component.ts [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import {
-  Component
+  Component, ChangeDetectionStrategy
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/list-toolbar/list-toolbar-search-actions.component.ts]: Imports need to be on their own lines.

File: src/app/public/modules/list-toolbar/list-toolbar.component.scss (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/list-toolbar/list-toolbar.component.scss]: Much better!

File: src/app/public/modules/list-toolbar/list-toolbar-search-actions.component.ts (hunk 3 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/list-toolbar/list-toolbar-search-actions.component.ts]: Whoops! Done.

**Pull request merged**